app/controllers/CashAccountController.scala

```diff
@@ -56,6 +58,11 @@ class CashAccountController @Inject() (
       if (appConfig.isCashAccountV2FeatureFlagEnabled) {
         Future.successful(Redirect(routes.CashAccountV2Controller.showAccountDetails(page)))
       } else {
+
+        cache.clear(request.eori).recover { case e =>
+          logger.error(s"Cache clear failed for ${request.eori}: ${e.getMessage}")
```

Comment:
We can't log EORIs as it's PII based on this guidance https://docs.tax.service.gov.uk/mdtp-handbook/documentation/manage-observability-of-a-microservice/log-information-about-your-service.html#what-not-to-log